lib/normalize_title_filter.js

```diff
@@ -1,33 +1,32 @@
 "use strict";
 
+var HyperSwitch = require('hyperswitch');
+var HTTPError = HyperSwitch.HTTPError;
 var mwUtil = require('./mwUtil');
 var P = require('bluebird');
 
 module.exports = function(hyper, req, next, options, specInfo) {
     var rp = req.params;
     /**
-     * Temporary logging & handling for rest.wikimedia.org requests.
+     * Temporary hint response for rest.wikimedia.org requests.
      */
     if (req.headers.host && req.headers.host === 'rest.wikimedia.org') {
-        // 1:10 sampled logging until we know that the rate is really low.
-        if (Math.random() < 0.1) {
-            hyper.log('warn/normalize/rest_wikimedia_org', {
-                msg: 'Request to rest.wikimedia.org',
-            });
-        }
-        var origNext = next;
-        // Make sure no rest.wikimedia.org requests are cached, as they aren't
-        // purged in Varnish frontends.
-        next = function(hyper, req) {
-            return origNext(hyper, req)
-            .then(function(res) {
-                if (res) {
-                    res.headers = res.headers || {};
-                    res.headers['cache-control'] = 'no-cache';
-                }
-                return res;
-            });
-        };
+        hyper.log('warn/normalize/rest_wikimedia_org', {
+            msg: 'Request to rest.wikimedia.org',
+        });
+
+        // Satisfy jscs's 100 char limit
+        var mURL = 'https://lists.wikimedia.org/pipermail/mediawiki-api/2016-January/003691.html';
+        throw new HTTPError({
+            status: 410,
+            body: {
+                type: 'not_found',
```

Comment:
I originally used not_found, as we already have an error page for that at https://www.mediawiki.org/wiki/HyperSwitch/errors/not_found. I agree that `gone` is more precise though, so I went ahead & created https://www.mediawiki.org/wiki/HyperSwitch/errors/gone / changed the type.